api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -55,7 +56,6 @@
 import org.pmiops.workbench.model.SendBillingSetupEmailRequest;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
-import org.springframework.util.StringUtils;
```

Comment:
this was the only instance of this import.  All others are like the replacement.

In this case, the only use was `capitalize()` which has the same behavior in both.